k8s/upgrades/0.8.0-0.8.1/cstor_volume_upgrade.sh

```diff
@@ -74,8 +78,8 @@ kubectl get deployment $c_dep -n $ns &>/dev/null
 rc=$?; if [ $rc -ne 0 ]; then echo "Target deployment not found: $rc"; exit; fi
 
 openebs_version=`kubectl get deployment $c_dep -n $ns -o jsonpath='{.metadata.labels.openebs\.io/version}'`
-if [ $openebs_version != "0.8.0" ]; then
-    echo "Current volumes version is not 0.8.0";exit 1;    
+if [ $openebs_version != $Current_version ]; then
```

Comment:
it will be good to exit if version of all targeted resources does not match `$current_version` or `$target_upgrade_version`

Reply:
yeah tried to do the same in new commit